### PR TITLE
Xz/default embedding

### PR DESCRIPTION
**Pull request by zhangxu0307**
- change default embedding model to sentence-transformers
- add depedency to requirement.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,3 +24,7 @@ colorama>=0.4.6
 
 # UI
 chainlit==0.7.700
+
+# Plugin Selection
+sentence-transformers>=2.2.2
```
Comment (liqul) [inline requirements.txt:29]: A bit heavy to install this? How about make it optional, and only if auto-selection is turned on, we ask the user to install this?

Comment (zhangxu0307) [reply requirements.txt:29]: Just implemented in [here](https://github.com/microsoft/TaskWeaver/blob/53cbfc9b15f5d21ebabcd98e424189464c0955b5/taskweaver/llm/sentence_transformer.py#L39C9-L47C14). Is it good and we do not need add the requirement?

Comment (liqul) [reply requirements.txt:29]: I support removing it from requiremnets

Comment (ShilinHe) [reply requirements.txt:29]: For UI chainlit requirement, it was added by a contributor. We can remove it and add the explanation to the ReadME (have done that for UI part).

Comment (zhangxu0307) [reply requirements.txt]: removed

**Review by Jack-Q — APPROVED** [review]

**Pull request merged**